manifests/importer-pod.yaml

```diff
@@ -7,45 +7,44 @@ spec:
   restartPolicy: Never
   containers:
   - name: importer
-    image: docker.io/jcoperh/importer:jcope-latest 
+    image: docker.io/jcoperh/importer:latest
     imagePullPolicy: Always
```

Comment:
not IfNotPresent? Or is it never present?

Reply:
we always want the latest, at least for now, because we're testing changes